sheet_cleaner/objects.py

```diff
@@ -228,50 +230,83 @@ def index2A1(self, num: int) -> str:
             raise ValueError('Could not convert index "{}" to A1 notation'.format(num))
 
 
-    def new_sheet(self, config, config_file_name):
-        '''
-        Make new spreadsheet as copy of itself (columns, no data). 
-        + Add new sheet info to config file
-        '''
 
-        service = build('sheets', 'v4', credentials=self.Auth(), cache_discovery=False)
-        request = service.spreadsheets().get(spreadsheetId=self.spreadsheetid, fields='properties.title')
-        response = request.execute()
-        title = response['properties']['title']
-        spreadsheet = {
-            'properties': {'title': title},
-            'sheets': {
-                'properties': {
-                    'sheetId': 0, 
-                    'title': self.name
-                },
-                'data': { 
-                    'startRow' : 0,
-                    'startColumn' : 0,
-                    'rowData': {
-                        'values': [{'userEnteredValue': {'stringValue': c}} for c in self.columns]
+class Template(GoogleSheet):
+    def __init__(self, spreadsheetid, token, credentials, is_service_account): 
+        self.spreadsheetid = spreadsheetid
```

Comment:
call `super().__init__(spreadsheetid, token, credentials, is_service_account)`
instead of duplicating the init code